content/en/blog/_posts/2016-10-00-Helm-Charts-Making-It-Simple-To-Package-And-Deploy-Apps-On-Kubernetes.md

```diff
@@ -57,7 +57,7 @@ The following applications are now available:
 
 
 1. 1.[Install Helm](https://github.com/kubernetes/helm/blob/master/docs/quickstart.md#install-helm)
-2. 2.[Initialize Helm](https://github.com/kubernetes/helm/blob/master/docs/quickstart.md#install-an-example-chart)
+2. 2.[Initialize Helm](https://github.com/helm/helm/blob/dev-v2/docs/quickstart.md)
```

Comment:
```suggestion
2. 2.[Initialize Helm](https://helm.sh/docs/intro/quickstart)
```

The dev branch will go away.  😬  In this case I'd either link to the official docs page or the last commit that had the file.

To get the last commit, you can clone the repo and use `git log --follow` to see the commits that touched the file.
```
[bob@muninn tmp]$ git clone git@github.com:helm/helm.git
Cloning into 'helm'...
remote: Enumerating objects: 6, done.
remote: Counting objects: 100% (6/6), done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 42061 (delta 0), reused 0 (delta 0), pack-reused 42055
Receiving objects: 100% (42061/42061), 13.66 MiB | 13.26 MiB/s, done.
Resolving deltas: 100% (28112/28112), done.
[bob@muninn tmp]$ cd helm
[bob@muninn helm] (master)$ git log --pretty="%H %s" --follow -- docs/quickstart.md
4f6d002d6c07d379508b5bfe53924753a3b1d4da chore(docs): move docs to helm-www
2571dbf82feb2697c9434bd6b2a69c4491b74100 ref: remove pkg/helm, pkg/hapi, pkg/tiller
79f88dfc5e54c786cd2064cc60992f6dd9905c47 ref(url) update helm's github url (#4962)
82c154e2ae13d1f175872c955b84ea92e5f96578 doc(helm): remove Tiller reference from the docs (#4788)
195d21d5d7a3e6be45065ddc7f5a9a7677fb4c93 ref(cmd): rename `helm delete` to `helm uninstall`
6029f4dc9325f436a2112fc4afa717329a2dd430 Fix helm ls in Quickstart
9dd74690c2d608b0058c8cd32403ccfb25c2d648 Updating the guidance for securing your Helm installation (#3228)
c20ec8fedaef10e0642d20bfe554baeb932aa0ee fixed bad link.
79ccafde2b4bc26829054fbbef9ca0c99f6ecd61 docs(quickstart): add clarifications about RBAC, TLS (#3180)
70f4f702e3f006d19c416020e27f788e35f61fd8 docs(install): document new upgrade process
0c8cff47aed21b333059f41268af334a2bdce15b Typo in helm quickstart guide.
627c59d148cd690910d512d13a9f7ef89ddaed1e docs(*): refresh docs
1c6fc9c0e857c1876f310bbbd6678134a00566ef feat(helm): remove the requirement that fetch/install need version
f51c7290468772af2dda0d9e79eeab60e86d5e34 fix for links to docs/examples in the quickstart
fed862985e77a8e65890f5b776dfc86108f9bb68 add docs table of contents and update quick start
a5eb5d7e0342040157bf7f7deb4f05824d655744 fix(docs): fix typo
2bbdff9e8ff82596c4170fd7ac01c0d8066e0b45 Update quickstart.md
9e060b0449763d2bb8ad20b75dab525f8b2f2ad7 docs(*): fixed minor issues with the docs.
e0eb95be8625f4a0d0ffe35e887719b14c4821c0 docs(*): add arch, charts, and quickstart doc
```

In this case we'd want the 2nd to last commit (`2571dbf82feb2697c9434bd6b2a69c4491b74100`) since the last one removed the file. From there you can use it in a link like this:
https://github.com/helm/helm/blob/2571dbf82feb2697c9434bd6b2a69c4491b74100/docs/quickstart.md